New API for retrieving messages from scylla.

kickhead13 opened the pull request:
The new api (.../get_server_messages_migration) can order, limit and offset messages from the backend.

Previously this was handled in the frontend (yes all of it: order + limit + offset)... now this is no longer needed.

AlexDuna reviewed (commented):
Code looks very well thought out.
Some observations i would make:
- Don't forget to add the get_channel_messages_migration API to our API docs.
- If there's no specific reason for all those functions to be inside db/mod.rs, it would be nice to create a messages.rs for specific message related functions inside db/ directory, just like in api/ directory, and also for the rest of user related functions, a separated users.rs file.

kickhead13 commented:
@AlexDuna I think i fixed the issues:
- commited changes to the db module structure (moved functions accordingly to their logical sub-modules)
- added the new API to the docs in this commit: https://github.com/onlinedi-vision/onlinedi-vision.github.io/commit/2c6eb0f1cbc5593911ad4bcebbca3e93b937cd76
